staging/src/k8s.io/client-go/examples/third-party-resources/main.go

```diff
@@ -165,3 +186,87 @@ func configureClient(config *rest.Config) {
 	metav1.AddToGroupVersion(scheme.Scheme, groupversion)
 	schemeBuilder.AddToScheme(scheme.Scheme)
 }
+
+func watch(client *rest.RESTClient) {
```

Comment:
I kind of think this should be in it's own file, and exported in case folks crib.